esp32_s2_kaluga_kit/esp32_s2_kaluga_kit.c

```diff
@@ -29,7 +37,7 @@ static const touch_pad_t bsp_touch_button[TOUCH_BUTTON_NUM] = {
     TOUCH_BUTTON_GUARD,      /*!< Guard ring for waterproof design. If this pad is touched, other pads no response.*/
 };
 
-const button_config_t bsp_button_config[BSP_BUTTON_NUM] = {
+static const button_config_t bsp_button_config[BSP_BUTTON_NUM] = {
     {
         .type = BUTTON_TYPE_ADC,
         .adc_button_config.adc_channel = ADC_CHANNEL_5, // ADC1 channel 5 is GPIO6
```

Comment:
In IDF5, there should be ADC initialized for keep the ADC handler for other use - e.g. https://github.com/espressif/esp-bsp/blob/a1844b9ace63719a8f8fc3b33a5c187fc165f605/esp32_s3_korvo_2/esp32_s3_korvo_2.c#L48

Reply:
There are some ugly changes required for this.
In Kaluga, we don't use the ADC for any other measurement.
Would it be OK if the user used the second ADC for his measurements?

Reply:
We was solving it here: https://github.com/espressif/esp-bsp/pull/201#pullrequestreview-1556913103 Now, the ADC is blocked by buttons and cannot be used other channels of the same ADC unit. After change, user can use it.